src/main.c

```diff
@@ -112,6 +145,16 @@ main() {
   /* Populate the USB device list */
   udev_fill_devices();
 
+  ret = xsdev_watch_init();
```

Comment:
Should this and xsdev_fill only be initialized in stub-mode?

Reply:
`xsdev_watch_init()` only registers the watch when `usb_backend_domid > 0`.  So it only runs in dom0/full-mode when a usbvm is detected.

Reply:
Gotcha. Thanks.